app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -113,6 +118,24 @@ class AdministratorControlsActivityTest {
   @Inject
   lateinit var context: Context
 
+  @get:Rule
+  val activityTestRule = ActivityTestRule(
+    AdministratorControlsActivity::class.java,
+    /* initialTouchMode= */ true,
+    /* launchActivity= */ false
+  )
+
+  @Test
+  fun testAdministratorControls_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(createAdministratorControlsActivityIntent(internalProfileId))
+    val title = activityTestRule.activity.title
+
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+
```

Comment:
Remove this blank line.

Reply:
Removed blank line: https://github.com/oppia/oppia-android/pull/3152/commits/3ee6278ea4eb85cd5f8ebae54ba6e1125b9bd362